lib/agent/socket_util.js

```diff
@@ -86,10 +86,11 @@ exports.secureSocket = function (socket, host, agent, mock)
     {
       if (!socket.authorized)
       {
-        return socket;
+        Logger.getInstance().error('Socket is not authorized: %s', socket.authorizationError);
```

Comment:
we are renewing the socket then? if so the it should be warn, if not close this comment

Reply:
Changed to warn.